src/Games/NexusMods.Games.RedEngine/Cyberpunk2077/SortOrder/RedMod/RedModSortableItemProviderFactory.cs

```diff
@@ -30,7 +30,8 @@ public class RedModSortableItemProviderFactory : ISortableItemProviderFactory
 
     public string IndexColumnHeader => "Load Order";
 
-    public string NameColumnHeader => "REDmod Name";
+    public string DisplayNameColumnHeader => "REDmod Name";
+    public string ParentModNameColumnHeader => "Parent mod";
```

Comment:
This could probably just be hardcoded in the column definition instead of being here. I don't know hoe likely it is to change for different games.